pkg/api/validate/dynamic/dynamic.go

```diff
@@ -87,13 +94,15 @@ func NewValidator(
 	authorizerType AuthorizerType,
 	cred azcore.TokenCredential,
 	pdpClient remotepdp.RemotePDPClient,
-) (Dynamic, error) {
+	byoNSG bool,
+) (*dynamic, error) {
```

Comment:
why is this change needed?

Reply:
Same as above.  It's another layer that I need to propagate the result flag from the validation.

Reply:
This has also been removed in the latest commit.